.eslintrc

```diff
@@ -25,11 +25,87 @@
     }]
   },
   "globals": {
+    "HTMLAllCollection": true,
```

Comment:
@LucaMele 
We should double check which native constructors we really need.
I would argue that all `HTML*Element` constructors are sufficient.